Backend enpoint override description update (#1063)
3scale/APIcast: doc/parameters.md

### `BACKEND_ENDPOINT_OVERRIDE`

> URI that overrides the backend endpoint. By default, it is the external route. 

Reviewer: I'm not sure it's a good idea to focus so much on Openshift given that this is valid for any kind of deployment.

Reviewer 2: That's the whole point of this PR as I see it. Could you please share what other use cases there are for this parameter then? Also if it's not on OpenShift I don't see when else this would be used.

Reviewer 3: Local development for example. Or exposing 3scale backend via a different route. Using geo routing, ...

### `BACKEND_ENDPOINT_OVERRIDE`

> URI that overrides the backend endpoint from the configuration, by default it is the external route. Is is useful when deploying APIcast into the same OpenShift cluster than 3scale as to use the internal hostname of the backend listener service instead of the public route. 

Reviewer 2: > ...by default it is the public openshift route. 
>...same OpenShift cluster as 3scale backend and use the internal service hostname of `backend-listener`.

### `BACKEND_ENDPOINT_OVERRIDE`

> URI that overrides the backend endpoint from the configuration, by default it is the external route. Is is useful when deploying APIcast into the same OpenShift cluster than 3scale as to use the internal hostname of the backend listener service instead of the public route. 

Reviewer 4: ```suggestion
URI that overrides the backend endpoint. By default, it is the external route. 
This parameter is useful when deploying APIcast into the same OpenShift cluster than 3scale, as when using the internal hostname of the backend listener service instead of the public route. 
```

URI that overrides the backend endpoint from the configuration, by default it is the external route. Is is useful when deploying APIcast into the same OpenShift cluster than 3scale as to use the internal hostname of the backend listener service instead of the public route. 

> **Example**: `http://backend-listener.<AMP_PROJECT>.svc.cluster.local:3000`.

Reviewer 2: `http://backend-listener.<3scale-namespace>.svc.cluster.local:3000`

Author: ```suggestion
**Example**: `http://backend-listener.<AMP_PROJECT>.svc.cluster.local:3000`.
**Example**: `http://backend-listener.<3scale-namespace>.svc.cluster.local:3000`
```
